app-crypt/keybase/keybase-1.0.48.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
why so many 1.4.x versions? It does not make any sense

Reply:
that said keep the latest one and remove the others

Reply:
ok